sensing/pointcloud_preprocessor/docs/ring-outlier-filter.md

```diff
@@ -22,11 +22,13 @@ This implementation inherits `pointcloud_preprocessor::Filter` class, please ref
 
 ### Core Parameters
 
-| Name                      | Type   | Default Value | Description |
-| ------------------------- | ------ | ------------- | ----------- |
-| `distance_ratio`          | double | 1.03          |             |
-| `object_length_threshold` | double | 0.1           |             |
-| `num_points_threshold`    | int    | 4             |             |
+| Name                      | Type    | Default Value | Description |
+| ------------------------- | ------- | ------------- | ----------- |
+| `distance_ratio`          | double  | 1.03          |             |
+| `object_length_threshold` | double  | 0.1           |             |
+| `num_points_threshold`    | int     | 4             |             |
+| `max_rings_num`           | uint_16 | 128           |             |
+| `max_points_num_per_ring` | size_t  | 2000          |             |
```

Comment:
How did you set the default value of `max_points_num_per_ring` as 2000 ?
Could I get the reason?

Reply:
It corresponds to figures that work well with the rosbag simulation in the AWF Tutorial. Are there any recommended default values? @Shin-kyoto

Reply:
@sykwer 
If there is no particular significance, I think it's best to set the value of `max_points_num_per_ring` according to the LiDAR. Let's set a value of 4000 as a large value. 
Also, please update the README and show that if this value is large enough such that FoV * resolution > `max_points_num_per_ring`, it is OK.